README.md

```diff
@@ -400,6 +402,43 @@ Unlike record elements, product element defintions must include identifiers.
 (product int_pair first::int second::int)
 ```
 
+#### Type Domain Includes
+
+It is possible to split type universes among multiple files, which allows type domains defined in another project to be
+permuted.  For example:
+
+```
+// root.ion:
+(include_file "sibling.ion")
+(include_file "sub-dir/thing.ion")
+(include_file "/other-project/toy-ast.ion")
+```
+
+While discussing further details, it is helpful to introduce two terms: an "includer" is a file which includes another
+using `include_file`, and the "includee" is a file which is included.
+
+The `root.ion` universe will contain all type domains from all includees and may still define additional type domains
+of its own.
+
+`include_file` statements are allowed in includees. Any attempt to include a file that has already been seen will be 
+ignored.
+
+When resolving the file to include, the following locations are searched:
+
+- The directory containing the includer (if the path to the includee does not start with `/`)
+- The directory containing the "main" type universe that was passed to `pig` on the command-line.
+- Any directories specified with the `--include` or `-I` arguments, in the order they were specified.
+
+The first matching file found wins and any other matching files ignored.
+
+Note that paths starting with `/` do not actually refer to the root of any file system, but instead are treated as 
+relative to the include directories.
```

Comment:
The prefix `/` here is basically a sigil modifying the include.

There are two slightly distinct types of inclusion:
- *unsigiled includes*
  -  include the includee relative to the 1) includer or 2) "main" or 3) any `-I` path
  - these serve to allow to break a model into more manageable bits
- *sigiled includes*: 
  - include the includee relative to any `-I` path
  - these feel more like referencing an external model

I see from the previous discussions on this issue that a formal idea of an `import` has been deferred, but I wonder if at least we should consider a sigil that is not as overloaded as `/`.